concepticondata/conceptlists/Gauchat-1925-480.tsv

```diff
@@ -0,0 +1,481 @@
+Gauchat-1925-480-384	384	I made myself	je me suis fait	sŭm factu	128		
+Gauchat-1925-480-385	385	a sprain	une entorse	intŏrsa	128		
+Gauchat-1925-480-386	386	I have had	J’ai eu	habūtu	128		
+Gauchat-1925-480-387	387	the fever	la fièvre	fĕbre	128	1534	FEVER
+Gauchat-1925-480-388	388	the doctor	Le médecin	mĕdicu	130	597	PHYSICIAN
+Gauchat-1925-480-389	389	had to	a dû	debūtu	130		
+Gauchat-1925-480-390	390	come	venir	venīre	130	1446	COME
+Gauchat-1925-480-391	391	He gave me	Il m’a donné	bajulatu	130		
+Gauchat-1925-480-392	392	a powder	une poudre	pŭlvere	130	3368	POWDER
+Gauchat-1925-480-393	393	white (feminine)	blanche	blanca	130		
+Gauchat-1925-480-394	394	yellow	jaune	galbina	132	1424	YELLOW
+Gauchat-1925-480-395	395	red	rouge	rŭbea	132	156	RED
+Gauchat-1925-480-396	396	green	verte	vĭrida	132	1425	GREEN
+Gauchat-1925-480-397	397	I took	J’ai pris	prē(hen)su	132		
+Gauchat-1925-480-398	398	a drop	une goutte	gŭtta	132	750	DROP
+Gauchat-1925-480-399	399	of gentian	de gentiane	gentiana	132		
+Gauchat-1925-480-400	400	(This) helped (me)	(Cela m’a) aidé	adjutatu	134		
+Gauchat-1925-480-401	401	to cure	à me guérir	warīre	134	499	CURE
+Gauchat-1925-480-402	402	Now	Maintenant	ad hōra	134	1376	NOW
+Gauchat-1925-480-403	403	I am cured	je suis guéri	sŭm warītu	134		
+Gauchat-1925-480-404	404	cured (feminine)	guérie	warīta	134		
+Gauchat-1925-480-405	405	(They are) healed (feminine)	(Elles sont) guéries	warītas	134		
+Gauchat-1925-480-406	406	he has pain	Il a mal	malu	136		
+Gauchat-1925-480-407	407	in the foot	au pied	pĕde	136		
+Gauchat-1925-480-408	408	(in the) knee	(au) genou	genŭclu	136	1371	KNEE
+Gauchat-1925-480-409	409	(in the) elbow	(au) coude	cŭbitu	136	981	ELBOW
+Gauchat-1925-480-410	410	(in the) finger	(au) doigt	dĭgitu	136	1303	FINGER
+Gauchat-1925-480-411	411	(in the) thumb	(au) pouce	pŏllice	136	1781	THUMB
+Gauchat-1925-480-412	412	(in the) nose	(au) nez	nasu	138	1221	NOSE
+Gauchat-1925-480-413	413	(in the) teeth	aux dents	dĕntes	138	1380	TOOTH
+Gauchat-1925-480-414	414	(in the) neck	(au) cou	collu	138	1333	NECK
+Gauchat-1925-480-415	415	(in the) heart	(au) cœur	cŏr	138	1223	HEART
+Gauchat-1925-480-416	416	(in the) tigh	(à la) cuisse	cŏxa	138	800	THIGH
+Gauchat-1925-480-417	417	(in one) rib	(à une) côté	cŏsta	138	801	RIB
+Gauchat-1925-480-418	418	in one eye	à un œil	ŏclu	140		
+Gauchat-1925-480-419	419	in the eyes	aux yeux	ŏclos	140		
+Gauchat-1925-480-420	420	(in the) tongue	(à la) langue	lĭngua	140	1205	TONGUE
+Gauchat-1925-480-421	421	(in the) head	(à la) tête	tĕsta	140	1256	HEAD
+Gauchat-1925-480-422	422	(in one) nail	(à un) ongle	ŭngla	140	1258	FINGERNAIL
+Gauchat-1925-480-423	423	(in one) ear	(à une) oreille	aurĭcla, -ŭcla	140	1247	EAR
+Gauchat-1925-480-424	424	He fell	Il est tombé	cadūtu, -ēctu	142		
+Gauchat-1925-480-425	425	(she) fell	(elle est) tombée	cadūta, -ēcta	142		
+Gauchat-1925-480-426	426	on his/her buttocks	sur son derrière	cūlu	142	834	BUTTOCKS
+Gauchat-1925-480-427	427	He falls	Il tombe	cadit	142		
```

Comment:
1280 FALL (or 2894 TUMBLE)

Reply:
I suspect the mapping strategy here was to attempt to map infinitives rather than inflectional verb forms. But since there is no other instance of 'fall', this one can be mapped.

Reply:
Yes, @Kristina-Pianykh. I think now, however, we may have less problems with our critical reviewers, if we unmap all cases with "il VERB", and really only leave infinitives, "the NOUN", and "a NOUN". Sorry that I did not see this earlier.

Reply:
Alright, will do.